developers/weaviate/client-libraries/python/v3_v4_migration.md

```diff
@@ -204,9 +224,9 @@ You can still create a collection from a JSON definition. This may be a useful w
 
 ### Removal of builder patterns
 
-The builder patterns for constructing queries have been removed, as they could be confusing and potentially lead to invalid queries.
+The builder patterns for constructing queries have been removed, as they could be confusing, and could lead to runtime errors that would not be picked up with static analysis.
```

Comment:
Consider:

The builder patterns for constructing queries have been removed. They could be confusing, and could lead to runtime errors that would not be picked up by static analysis.

Reply:
Updated